README.md

```diff
@@ -19,4 +19,117 @@ conda env create -y -f environment.yml && conda activate biogtr
+One of the main advantages of this system is that we ***decouple** detection and tracking. This means that our model assumes that you already have detections available when you are ready to track. In order to get these detections we recommend a couple methods. For animal pose-estimation, we highly recommend heading over to [`sleap`](https://sleap.ai) and running through their work flow. For microscopy tracking, checkout [`trackmate`](https://imagej.net/plugins/trackmate/) as well as [`cellpose`](https://www.cellpose.org) and [`StarDist`](https://imagej.net/plugins/stardist). If you are only doing tracking (ie inference), then you once you have your detections you are good to go! 
+### ***FOR TRAINING*** Step 1.2 Proofreading
+Otherwise, we recommend using the `sleap-label` gui to proofread your track labels. For animal tracking, if you used `sleap` you can start proofreading right away. Otherwise if you used a different system (e.g `DeepLabCut`) check out `sleap.io.format` for available converters. With microscopy, we highly recommend starting out with `trackmate` and then proofreading in `sleap`. Here is [a converter from trackmate's output to a `.slp`](https://gist.github.com/aaprasad/5243be0785a40e9dafa1697ce2258e3e) file. In general, you can use [`sleap-io`](https://io.sleap.ai/latest/) to write a custom converter to `.slp` if you'd like to use the sleap-gui for proofreading
+### Step 1.3 Organize data.
+
+Although, our data loading does take paths to label files and video files directly so it's fairly flexible, we recommend organizing your data with a couple things in mind.
+
+1. Match video and labels file stems. Because our datasets just take in a list of video files and a list of label files, the order in which the corresponding files are passed must match. (e.g `[file1.slp, file2.slp], [vid1.mp4, vid2.mp4]`). In order to make programmatic file searching easy its best to save your labels and vid files with the same stem so that you can ensure the ordering will be consistent. Its also just best practice so you know which video a labels file corresponds to.
+2. Store corresponding videos and labels files in the same directory. This will again make searching for the files much easier. 
+
+The best practice is to have your dataset organized as follows:
+```bash
+dataset_name/
+    train/
+        vid_1.{VID_EXTENSION}
+        vid_1.{LABELS_EXTENSION}
+        ...
+        vid_n.{VID_EXTENSION}
+        vid_n.{LABELS_EXTENSION}
+    val/
+        vid_1.{VID_EXTENSION}
+        vid_1.{LABELS_EXTENSION}
+        ...
+        vid_n.{VID_EXTENSION}
+        vid_n.{LABELS_EXTENSION}
+    test/
+        vid_1.{VID_EXTENSION}
+        vid_1.{LABELS_EXTENSION}
+        ...
+        vid_n.{VID_EXTENSION}
+        vid_n.{LABELS_EXTENSION}
+    inference/
+        vid_1.{VID_EXTENSION}
+        vid_1.{LABELS_EXTENSION} # these files don't need tracks
+        ...
+        vid_n.{VID_EXTENSION}
+        vid_n.{LABELS_EXTENSION}
```

Comment:
Put these in another section when you get to inference.

Also, do we really support any labels extension that isn't `.slp`? Either way, let's just be explicit.

We should also be explicit about the video extensions. Especially since we're doing microscopy, it would be good to know if zarr and multi-frame TIFF stacks are supported. Since `sleap-io` provides the layer for doing video reading, it's totally okay to redirect users to the [sleap-io docs](https://io.sleap.ai/latest/formats/#sleap_io.load_video). (I'll add more explicit info there.)